spec/features/user_can_view_past_orders_spec.rb

```diff
@@ -64,37 +37,24 @@
   end
 
   scenario "authenticated user cannot see other users orders" do
-    user = User.create(first_name: "default",
-                       last_name: "user",
-                       username: "default_user",
-                       password: "default",
-                       role: 0 )
-    user_2 = User.create(first_name: "default",
-                       last_name: "user",
-                       username: "default_user_2",
-                       password: "default",
-                       role: 0 )
-    order = user.orders.create(status: "Paid",
-                               total_price: 12)
-    order_2 = user.orders.create(status: "Pending",
-                                 total_price: 24)
-    order_3 = user_2.orders.create(status: "Cancelled",
-                                   total_price: 36)
-
-    visit login_path
+    user_1 = create(:user)
+    user_2 = create(:user)
+    travesty = create(:travesty_with_items)
+    item_1, item_2, item_3, item_4 = travesty.items
```

Comment:
Useless assignment to variable - `item_1`. Use `_` or `_item_1` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `item_2`. Use `_` or `_item_2` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `item_3`. Use `_` or `_item_3` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `item_4`. Use `_` or `_item_4` as a variable name to indicate that it won't be used.